src/actions/photos.js

```diff
@@ -0,0 +1,106 @@
+/**
+  Photos related features
+**/
+
+import cozy from 'cozy-client-js'
+
+import {
+  COZY_PHOTOS_DIR_ID,
+  FETCH_PHOTOS,
+  FETCH_PHOTOS_FAILURE,
+  RECEIVE_PHOTOS,
+  UPLOAD_PHOTOS,
+  UPLOAD_PHOTOS_SUCCESS,
+  UPLOAD_PHOTOS_SUCCESS_WITH_CONFLICTS,
+  UPLOAD_PHOTOS_FAILURE
+} from './constants'
```

Comment:
I second @gregorylegarec, I think it makes sense to regroup these constants with the action creators.